### PR TITLE
Package python client library in a single wheel file to upload on pyPI

### PR DESCRIPTION
**Pull request by tanmayv25**
…nager

These changes package our python client library into a monolithic design. Users can specify optional features (grpc, http or all), which will install the required dependencies. 

Look here for more details:
https://github.com/triton-inference-server/server/blob/0a67591c1bb4a7225a0399f8eec68ab536cccd9c/src/clients/python/library/README.md

We are supporting the previous package organization, but users will see a warning message. I have modified our examples to use the new package organization but qa still tests the older package distribution to ensure the test coverage. 

[Actually there are two different wheel files for separate paltform systems: generic and x86-linux ]

### CODE REVIEW

File: src/clients/python/library/README.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+# Triton Client Library
```
Comment (deadeyegoodwin) [inline src/clients/python/library/README.md:29]: Triton Python Client Library

**Comment by CoderHam** [comment]
@tanmayv25 L0_win10 will also have to be modified to test the new package format. You can do that in a follow up ticket along with other gitlab side changes.

**Comment by deadeyegoodwin** [comment]
@tanmayv25  is this still [WIP]?

**Comment by tanmayv25** [comment]
Jonathan gave some suggestions to further improve the structure. Currently, trying to fix the backwards compatibility with those changes. Hence WIP..

**Comment by DEKHTIARJonathan** [comment]
LGTM. Merged approved on my side

**Pull request merged**